extension/js/common/platform/store/contact-store.ts

```diff
@@ -194,10 +164,8 @@ export class ContactStore extends AbstractStore {
         throw new Error('contact not found right after inserting it');
       }
     }
-    if (update.pubkey && update.pubkey.includes(PgpArmor.headers('privateKey').begin)) { // wrongly saving prv instead of pub
-      Catch.report('Wrongly saving prv as contact - converting to pubkey');
-      const key = await PgpKey.read(update.pubkey);
-      update.pubkey = key.toPublic().armor();
+    if (update.pubkey) {
```

Comment:
Please add back the `      Catch.report('Wrongly saving prv as contact - converting to pubkey');` - this is meant to inform us so that we can fix such bugs, especially after large refactors

Reply:
Whops, sorry, removed by mistake.